public/scripts/actions.js

```diff
@@ -620,3 +621,11 @@ export function spinLogo() {
     type: SPIN_LOGO
   }
 }
+
+
+export function checkUncheckAll(prefix) {
```

Comment:
Don't call this uncheckAll---this is a generic actions file, so you have to specify what it does, e.g. checkAllFeatures and uncheckAllFeatures